src/librustc_resolve/lib.rs

```diff
@@ -444,9 +443,15 @@ fn resolve_error<'b, 'a: 'b, 'tcx: 'a>(resolver: &'b Resolver<'a, 'tcx>,
                                                    module = &*path,
                                                    ident = ident.node);
                             }
-
-                            _ => {} // no help available
+                            ExprCall(_, _) => {
+                                help_msg = format!("No function corresponds to `{module}(..)`",
+                                                   module = &*path);
+                            }
+                            _ => { } // no help available
                         }
+                    } else {
+                        help_msg = format!("Module `{module}` cannot be used as an object!",
```

Comment:
hmm, do we used the terminology "used as an object" elsewhere?  (That is supposed to mean that its been used as the receiver in a method call, right?)

On the one hand, your phrasing is succinct. But on the other hand, I'm not sure everyone will understand what it means ...